.ci.yaml

```diff
@@ -1073,6 +1071,15 @@ targets:
       version_file: flutter_master.version
       target_file: macos_check_podspecs.yaml
 
+  - name: Mac_arm64 repo_checks
+    recipe: packages/packages
+    timeout: 30
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
+      version_file: flutter_master.version
+      target_file: macos_check_podspecs.yaml
```

Comment:
I'll rename to `macos_repo_checks_podspecs.yaml` once `Mac_x64 check_podspecs` is removed and it's not shared.